include/d/d_tree.h

```diff
@@ -9,6 +9,22 @@ class cCcD_Obj;
 class dCcMassS_HitInf;
 class fopAc_ac_c;
 
+namespace d_tree {
+    extern u8 g_dTree_shadowPos[];
+    extern u8 g_dTree_shadowTexCoord[];
+
+    extern Vec l_pos[];
+    extern GXColor l_color[];
+    extern cXy l_texCoord[];
+    
+    extern const u32 g_dTree_shadowMatDL_SIZE;
+    extern const u32 g_dTree_Oba_kage_32DL_SIZE;
+
+    extern u8 l_matDL[];
+    extern u8 g_dTree_shadowMatDL[];
+    extern u8 g_dTree_Oba_kage_32DL[];
+};
+
```

Comment:
This shouldn't be necessary.

Reply:
Not necessary (I could use externs), but d_wood does use these symbols from the d_tree namespace (and so does d_tree). I figured it'd be cleaner to keep them in the header. 

Do you think I should just remove them from the header?

Reply:
d_tree isn't a namespace, it's just the object name. keep the extern declarations in the header but remove the namespace - the `g_` prefix means they're global variables.

Reply:
oh but the `l_` ones are local, so those should be removed from the header, yes.

Reply:
Ohhhh gotcha